DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/part1_native/src/use_native.cpp

```diff
@@ -37,33 +64,16 @@ int main() {
     std::cout << "Running on device: "
               << device.get_info<sycl::info::device::name>().c_str()
               << std::endl;
+    {
+      //write data to host-to-device hostpipe
+      InputPipeA::write(q, kA);
+      InputPipeB::write(q, kB);
+      //launch kernal that would compute multiplication as per compiler choice
```

Comment:
```suggestion
      // launch kernel that infers a multiplier automatically
```
please fix this spelling anywhere else